File: Actividades Lenguaje Clase 3/Actividad_4_Lp.py
import random

valores = [random.randint(50, 100) for _ in range(500)]

pares = sum(1 for v in valores if v % 2 == 0)
impares = 500 - pares

print(f"Valores pares: {pares}")
print(f"Valores impares: {impares}")

File: Actividades Lenguaje Clase 3/Actividad_4_Lp.cpp
#include <iostream>
#include <random>
int main() {
    
    int pares = 0;
    int impares = 0;

     for (int i = 0; i < 500; ++i) {
        int valor = 50 + std::rand() % 51;  
        if (valor % 2 == 0) {
            ++pares;
        } else {
            ++impares;
        }
    }

    std::cout << "Valores pares: " << pares << std::endl;
    std::cout << "Valores impares: " << impares << std::endl;

    return 0;
}